docs/extend/extend-desktop/mvd-dataservices.md

```diff
@@ -151,7 +151,7 @@ In the `zlux-app-server/zluxserver.json` file, use the example below to specify
           }
         }
       },
-      "portRange": [8545,8600]
+      "portRange": [7558,7613]
```

Comment:
To be honest it would be good to set this to like, 7600-7650. It's just a range of ports to use, but if we put it to close to the existing ports it could become overlapping by accident and make the documentation outdated.